symplyphysics/laws/chemistry/boundary_of_thermalization_zone_of_atomized_atoms_in_magnetron.py

```diff
@@ -0,0 +1,38 @@
+from sympy import Eq, solve
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output, dimensionless,)
+
+# Description
+## The atoms of the target material evaporate and move towards the substrate inside the magnetron. There is a distance at which the energy of the atomized
+## atom will decrease to the energy of the thermal motion of the atoms of the gas-discharge plasma. This distance is called the boundary of the thermalization zone.
+## The decrease in energy occurs due to the collision of the atomized atom with the gas atoms in the magnetron. The free path length is the distance that the sprayed
+## gas travels between two collisions.
+
+## Law is: L = N * l, where
+## L - boundary of thermalization zone of atomized atoms in magnetron,
+## N - the number of collisions of atomized atom with gas atoms (is a statistical quantity, can be neither an integer),
```

Comment:
It depends on time and distance traveled by atom. Can we define some bounds, where this parameter is estimated? For example stationary atom would get infinite amount of collisions at infinite time period.